src/index.ts

```diff
@@ -9,17 +9,19 @@ const handlers: { [key: string]: Handler } = {};
 const myName = process.env.name;
 const myPmId = Number(process.env.pm_id);
 let myBus: any;
+const globalResponseListeners: Record<string, any> = {};
```

Comment:
a key component of this solution is to store listeners in a global var. These can later on be removed.